src/dlgMapper.cpp

```diff
@@ -197,26 +197,22 @@ void dlgMapper::updateAreaComboBox()
     comboBox_showArea->setCurrentText(oldValue); // Try and reset to previous value
 }
 
-void dlgMapper::slot_toggleShowRoomIDs(int s)
+void dlgMapper::slot_toggleShowRoomIDs(int toggle)
 {
-    if (s == Qt::Checked) {
-        mp2dMap->mShowRoomID = true;
-    } else {
-        mp2dMap->mShowRoomID = false;
-    }
+    mp2dMap->mShowRoomID = toggle == Qt::Checked;
```

Comment:
Just for clarity I'd brace the test expression:
```suggestion
    mp2dMap->mShowRoomID = (toggle == Qt::Checked);
```

Reply:
Your action may be needed.

Reply:
Applied.